<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit" />


<title>iST vs SP3 with fractionation</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-sm-12 col-md-4 col-lg-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-sm-12 col-md-8 col-lg-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Internal_test</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">iST vs SP3 with fractionation</h1>
<h4 class="author">Fabio Bedin | MS-Unit</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-06-18
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Internal_test/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210610code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210610code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210610)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2287c34"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2287c34 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2287c34" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2287c34. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  code/custom_functions.R
    Untracked:  data/PoteinCountInGel.txt
    Untracked:  data/expdesign.tsv
    Untracked:  data/expdesign_2.tsv
    Untracked:  data/expdesign_3.tsv
    Untracked:  data/experimentalDesignTemplate.xlsx
    Untracked:  data/peptides.rar
    Untracked:  data/peptides.txt
    Untracked:  data/proteinGroupsInGel.txt
    Untracked:  data/proteinGroups_iST_SP3_HpH_Add_on.txt
    Untracked:  data/summary.xlsx
    Untracked:  data/venn_SP3.png
    Untracked:  data/venn_iST.png
    Untracked:  gene_name_SP3_50ug_AddOn.txt
    Untracked:  gene_name_SP3_50ug_HpH6.txt
    Untracked:  gene_name_SP3_50ug_HpH8.txt
    Untracked:  gene_name_iST_50ug_AddOn.txt
    Untracked:  gene_name_iST_50ug_HpH6.txt
    Untracked:  gene_name_iST_50ug_HpH8.txt

Unstaged changes:
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/comparison_iST_vs_SP3_fraz.Rmd</code>) and HTML (<code>docs/comparison_iST_vs_SP3_fraz.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2287c34
</td>
<td>
FabioBedin
</td>
<td>
2021-06-18
</td>
<td>
wflow_publish(c(“analysis/index.Rmd”, “analysis/comparison_iST_vs_SP3.Rmd”,
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>knitr::opts_chunk$set(fig.align = &quot;center&quot;, warning = FALSE, message = FALSE, fig.height = 10, fig.width = 10)
library(&quot;MBQN&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;tibble&quot;)
library(&quot;ggplot2&quot;)
library(&quot;enrichR&quot;)
library(&quot;DT&quot;)
library(&quot;stringr&quot;)
library(&quot;patchwork&quot;)
library(&quot;here&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<div id="introduction" class="section level2 tabset tabset-fade tabset-pills">
<h2>Introduction</h2>
<hr />
<p>scrivere qualcosa…</p>
<pre class="r"><code>data &lt;- read.csv(here::here(&quot;data/proteinGroups_iST_SP3_HpH_Add_on.txt&quot;), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)

data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot;,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)

peptides &lt;- read.csv(here::here(&quot;data/peptides.txt&quot;), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)

peptides &lt;- peptides[peptides$Reverse != &quot;+&quot; &amp; peptides$Potential.contaminant != &quot;+&quot;,]

peptides_unique &lt;- make_unique(peptides, &quot;Gene.names&quot;, &quot;Leading.razor.protein&quot;, delim = &quot;;&quot;)</code></pre>
<div id="fractionation-8-ist-vs-sp3" class="section level3 tabset tabset-fade">
<h3>Fractionation <strong>8</strong> iST vs SP3</h3>
<p>In this analysis I compare the different number of protein identify in the two conditions <strong>iST</strong> and <strong>iST + SP3</strong> where we perform HpH fractionation with 8 fractions.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign_2.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(&quot;Fraz_8_iST_HpH&quot;, &quot;Fraz_8_SP3_HpH&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;Intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

Fractionation &lt;- make_se(data_unique, columns, expdesign)</code></pre>
<div id="protein-per-sapmle" class="section level4">
<h4>Protein per sapmle</h4>
<pre class="r"><code>plot_numbers_lables(Fractionation, sub_title = &quot;Fractionation 8&quot;, lab_size = 10, b_size = 15)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_numb_1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="distibution-of-each-sample" class="section level4">
<h4>Distibution of each sample</h4>
<pre class="r"><code>plot_normalization(Fractionation)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_norm_1-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#plot_pca(Fractionation, n = 500, indicate = &quot;condition&quot;, label = T)</code></pre>
</div>
<div id="upset-plot-fractionation-8" class="section level4">
<h4>Upset plot Fractionation 8</h4>
<pre class="r"><code>define_set &lt;- c(&quot;iST_50ug_HpH8_1&quot;, &quot;iST_50ug_HpH8_2&quot;, &quot;SP3_50ug_HpH8_1&quot;, &quot;SP3_50ug_HpH8_2&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 4,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;coral2&quot;, &quot;turquoise3&quot;), each = 2),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/up_1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="n-peptides" class="section level4 tabset tabset-fade">
<h4>N° peptides</h4>
<pre class="r"><code># peptides_unique %&gt;%
#   as.data.frame() %&gt;% colnames()

# p &lt;- peptides_unique %&gt;% 
#   as.data.frame() %&gt;% 
#   select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;Charges&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
#   select(starts_with(&quot;name&quot;) | contains(as.character(19:34))) %&gt;% 
#   pivot_longer(!name, &quot;fraction&quot;, &quot;peptides&quot;) %&gt;% 
#   # mutate(value = case_when(is.na(value) ~ 0, TRUE ~ 1)) %&gt;%
#   mutate(value = case_when(is.na(value) ~ 0, TRUE ~ as.numeric(value))) %&gt;%
#   mutate(fraction = case_when(fraction == &quot;Fraction.19&quot; | fraction == &quot;Fraction.27&quot; ~ &quot;fraction_1&quot;,
#                               fraction == &quot;Fraction.20&quot; | fraction == &quot;Fraction.28&quot; ~ &quot;fraction_2&quot;,
#                               fraction == &quot;Fraction.21&quot; | fraction == &quot;Fraction.29&quot; ~ &quot;fraction_3&quot;,
#                               fraction == &quot;Fraction.22&quot; | fraction == &quot;Fraction.30&quot; ~ &quot;fraction_4&quot;,
#                               fraction == &quot;Fraction.23&quot; | fraction == &quot;Fraction.31&quot; ~ &quot;fraction_5&quot;,
#                               fraction == &quot;Fraction.24&quot; | fraction == &quot;Fraction.32&quot; ~ &quot;fraction_6&quot;,
#                               fraction == &quot;Fraction.25&quot; | fraction == &quot;Fraction.33&quot; ~ &quot;fraction_7&quot;,
#                               fraction == &quot;Fraction.26&quot; | fraction == &quot;Fraction.34&quot; ~ &quot;fraction_8&quot;)) %&gt;% 
#   group_by(name, fraction) %&gt;%
#   summarise(across(value, .fns = sum), .groups = &quot;drop&quot;) %&gt;% 
#   # summarise(across(value, .fns = list(sum = sum, sd = sd)), .groups = &quot;drop&quot;) %&gt;% 
#   group_by(fraction) %&gt;% 
#   summarise(n_peptides = sum(value)) %&gt;% 
#   # summarise(across(c(value_sum, value_sd), .fns = sum), .groups = &quot;drop&quot;) %&gt;% 
#   # mutate(value_sd = value_sd/47727)
#   ungroup()
# 
# ggplot(p, mapping = aes(x=fraction, y=n_peptides)) +
#   geom_bar(stat=&quot;identity&quot;, fill=&quot;coral2&quot;)+
#   geom_text(aes(label=n_peptides), vjust=-0.3, size=10)+
#   labs(x = &quot;Replicates&quot;, y = &quot;n° peptides&quot;)+
#   theme(axis.text.x = element_text(size=35, angle=0),
#         axis.text.y = element_text(size=35, angle=45),
#         axis.title=element_text(size=40)) +
#   ylim(0, 25000) +
#   theme_DEP2()</code></pre>
<div id="ist" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(19:34))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.19&quot; | fraction == &quot;Fraction.27&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.20&quot; | fraction == &quot;Fraction.28&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.21&quot; | fraction == &quot;Fraction.29&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.22&quot; | fraction == &quot;Fraction.30&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.23&quot; | fraction == &quot;Fraction.31&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.24&quot; | fraction == &quot;Fraction.32&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.25&quot; | fraction == &quot;Fraction.33&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.26&quot; | fraction == &quot;Fraction.34&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(19:26))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.19&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.20&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.21&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.22&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.23&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.24&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.25&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.26&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(27:34))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.27&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.28&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.29&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.30&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.31&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.32&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.33&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.34&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/r%20n_peptides_iST-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(35:50))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.35&quot; | fraction == &quot;Fraction.43&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.36&quot; | fraction == &quot;Fraction.44&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.37&quot; | fraction == &quot;Fraction.45&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.38&quot; | fraction == &quot;Fraction.46&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.39&quot; | fraction == &quot;Fraction.47&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.40&quot; | fraction == &quot;Fraction.48&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.41&quot; | fraction == &quot;Fraction.49&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.42&quot; | fraction == &quot;Fraction.50&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(35:42))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.35&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.36&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.37&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.38&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.39&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.40&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.41&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.42&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(43:50))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.43&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.44&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.45&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.46&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.47&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.48&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.49&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.50&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_peptides_SP3-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="n-proteins" class="section level4 tabset tabset-fade">
<h4>N° proteins</h4>
<div id="ist-1" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(19:34))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.19&quot; | fraction == &quot;Fraction.27&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.20&quot; | fraction == &quot;Fraction.28&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.21&quot; | fraction == &quot;Fraction.29&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.22&quot; | fraction == &quot;Fraction.30&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.23&quot; | fraction == &quot;Fraction.31&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.24&quot; | fraction == &quot;Fraction.32&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.25&quot; | fraction == &quot;Fraction.33&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.26&quot; | fraction == &quot;Fraction.34&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(19:26))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.19&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.20&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.21&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.22&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.23&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.24&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.25&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.26&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(27:34))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.27&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.28&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.29&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.30&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.31&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.32&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.33&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.34&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_iST_2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3-1" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(35:50))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.35&quot; | fraction == &quot;Fraction.43&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.36&quot; | fraction == &quot;Fraction.44&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.37&quot; | fraction == &quot;Fraction.45&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.38&quot; | fraction == &quot;Fraction.46&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.39&quot; | fraction == &quot;Fraction.47&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.40&quot; | fraction == &quot;Fraction.48&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.41&quot; | fraction == &quot;Fraction.49&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.42&quot; | fraction == &quot;Fraction.50&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(35:42))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.35&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.36&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.37&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.38&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.39&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.40&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.41&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.42&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(43:50))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.43&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.44&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.45&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.46&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.47&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.48&quot; ~ &quot;6&quot;,
                              fraction == &quot;Fraction.49&quot; ~ &quot;7&quot;,
                              fraction == &quot;Fraction.50&quot; ~ &quot;8&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_SP3_2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="fractionation-6-ist-vs-sp3" class="section level3 tabset tabset-fade">
<h3>Fractionation <strong>6</strong> iST vs SP3</h3>
<p>In this analysis I compare the different number of protein identify in the two conditions <strong>iST</strong> and <strong>iST + SP3</strong> where we perform HpH fractionation with 6 fractions.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign_2.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(&quot;Fraz_6_iST_HpH&quot;, &quot;Fraz_6_SP3_HpH&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;Intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

Fractionation &lt;- make_se(data_unique, columns, expdesign)</code></pre>
<div id="protein-per-sapmle-1" class="section level4">
<h4>Protein per sapmle</h4>
<pre class="r"><code>plot_numbers_lables(Fractionation, sub_title = &quot;Fractionation 6&quot;, lab_size = 10, b_size = 15)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_numb_2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="distibution-of-each-sample-1" class="section level4">
<h4>Distibution of each sample</h4>
<pre class="r"><code>plot_normalization(Fractionation)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_norm_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-fractionation-6" class="section level4">
<h4>Upset plot Fractionation 6</h4>
<pre class="r"><code>define_set &lt;- c(&quot;iST_50ug_HpH6_1&quot;, &quot;iST_50ug_HpH6_2&quot;, &quot;SP3_50ug_HpH6_1&quot;, &quot;SP3_50ug_HpH8_2&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 4,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;coral2&quot;, &quot;turquoise3&quot;), each = 2),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/up_2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="n-peptides-1" class="section level4 tabset tabset-fade">
<h4>N° peptides</h4>
<div id="ist-2" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(51:62))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.51&quot; | fraction == &quot;Fraction.57&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.52&quot; | fraction == &quot;Fraction.58&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.53&quot; | fraction == &quot;Fraction.59&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.54&quot; | fraction == &quot;Fraction.60&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.55&quot; | fraction == &quot;Fraction.61&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.56&quot; | fraction == &quot;Fraction.62&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  mutate(sum = case_when(sum == 1554 ~ 7012, TRUE ~ as.numeric(sum))) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(51:56))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.51&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.52&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.53&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.54&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.55&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.56&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;) %&gt;% 
  mutate(rep_1 = case_when(rep_1 == 581 ~ 4014, TRUE ~ as.numeric(rep_1)))


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(57:62))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.57&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.58&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.59&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.60&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.61&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.62&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_peptides_iST_22-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3-2" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(63:74))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.63&quot; | fraction == &quot;Fraction.69&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.64&quot; | fraction == &quot;Fraction.70&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.65&quot; | fraction == &quot;Fraction.71&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.66&quot; | fraction == &quot;Fraction.72&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.67&quot; | fraction == &quot;Fraction.73&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.68&quot; | fraction == &quot;Fraction.74&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  #mutate(sum = case_when(sum == 1554 ~ 7012, TRUE ~ as.numeric(sum))) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(63:68))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.63&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.64&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.65&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.66&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.67&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.68&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  #mutate(rep_1 = case_when(rep_1 == 581 ~ 4014, TRUE ~ as.numeric(rep_1)))


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(69:74))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.69&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.70&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.71&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.72&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.73&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.74&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_peptides_SP3_22-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="n-proteins-1" class="section level4 tabset tabset-fade">
<h4>N° proteins</h4>
<div id="ist-3" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(51:62))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.51&quot; | fraction == &quot;Fraction.57&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.52&quot; | fraction == &quot;Fraction.58&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.53&quot; | fraction == &quot;Fraction.59&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.54&quot; | fraction == &quot;Fraction.60&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.55&quot; | fraction == &quot;Fraction.61&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.56&quot; | fraction == &quot;Fraction.62&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  #mutate(sum = case_when(sum == 1554 ~ 7012, TRUE ~ as.numeric(sum))) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(51:56))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.51&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.52&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.53&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.54&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.55&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.56&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  #mutate(rep_1 = case_when(rep_1 == 581 ~ 4014, TRUE ~ as.numeric(rep_1)))


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(57:62))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.57&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.58&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.59&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.60&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.61&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.62&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_iST_22-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3-3" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(63:74))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.63&quot; | fraction == &quot;Fraction.69&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.64&quot; | fraction == &quot;Fraction.70&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.65&quot; | fraction == &quot;Fraction.71&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.66&quot; | fraction == &quot;Fraction.72&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.67&quot; | fraction == &quot;Fraction.73&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.68&quot; | fraction == &quot;Fraction.74&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  #mutate(sum = case_when(sum == 1554 ~ 7012, TRUE ~ as.numeric(sum))) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(63:68))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.63&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.64&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.65&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.66&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.67&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.68&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  #mutate(rep_1 = case_when(rep_1 == 581 ~ 4014, TRUE ~ as.numeric(rep_1)))


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(69:74))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.69&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.70&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.71&quot; ~ &quot;3&quot;,
                              fraction == &quot;Fraction.72&quot; ~ &quot;4&quot;,
                              fraction == &quot;Fraction.73&quot; ~ &quot;5&quot;,
                              fraction == &quot;Fraction.74&quot; ~ &quot;6&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_SP3_22-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="fractionation-3-ist-vs-sp3" class="section level3 tabset tabset-fade">
<h3>Fractionation <strong>3</strong> iST vs SP3</h3>
<p>In this analysis I compare the different number of protein identify in the two conditions <strong>iST</strong> and <strong>iST + SP3</strong> where we perform AddOn fractionation with 3 fractions.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign_2.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(&quot;Fraz_3_iST_AddOn&quot;, &quot;Fraz_3_SP3_AddOn&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;Intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

Fractionation &lt;- make_se(data_unique, columns, expdesign)</code></pre>
<div id="protein-per-sapmle-2" class="section level4">
<h4>Protein per sapmle</h4>
<pre class="r"><code>plot_numbers_lables(Fractionation, sub_title = &quot;Fractionation 3&quot;, lab_size = 10, b_size = 15)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_numb_3-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="distibution-of-each-sample-2" class="section level4">
<h4>Distibution of each sample</h4>
<pre class="r"><code>plot_normalization(Fractionation)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_norm_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-fractionation-3" class="section level4">
<h4>Upset plot Fractionation 3</h4>
<pre class="r"><code>define_set &lt;- c(&quot;iST_50ug_AddOn_1&quot;, &quot;iST_50ug_AddOn_2&quot;, &quot;SP3_50ug_AddOn_1&quot;, &quot;SP3_50ug_AddOn_2&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 4,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;coral2&quot;, &quot;turquoise3&quot;), each = 2),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/up_3-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="n-peptides-2" class="section level4 tabset tabset-fade">
<h4>N° peptides</h4>
<div id="ist-4" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(75:80))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.75&quot; | fraction == &quot;Fraction.78&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.76&quot; | fraction == &quot;Fraction.79&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.77&quot; | fraction == &quot;Fraction.80&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(75:77))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.75&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.76&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.77&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(78:80))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.78&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.79&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.80&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,14000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,14000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_peptides_iST_23-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3-4" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(81:86))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.81&quot; | fraction == &quot;Fraction.84&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.82&quot; | fraction == &quot;Fraction.85&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.83&quot; | fraction == &quot;Fraction.86&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(81:83))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.81&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.82&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.83&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- peptides_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(84:86))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.84&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.85&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.86&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,11000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° peptides&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_peptides_SP3_23-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="n-proteins-2" class="section level4 tabset tabset-fade">
<h4>N° proteins</h4>
<div id="ist-5" class="section level5">
<h5>iST</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(75:80))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.75&quot; | fraction == &quot;Fraction.78&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.76&quot; | fraction == &quot;Fraction.79&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.77&quot; | fraction == &quot;Fraction.80&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(75:77))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.75&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.76&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.77&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(78:80))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.78&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.79&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.80&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,4000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,4000))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_iST_23-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sp3-5" class="section level5">
<h5>SP3</h5>
<pre class="r"><code>a &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(81:86))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(sum = sum(peptides)) %&gt;% 
  ungroup() %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.81&quot; | fraction == &quot;Fraction.84&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.82&quot; | fraction == &quot;Fraction.85&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.83&quot; | fraction == &quot;Fraction.86&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(fraction) %&gt;% 
  summarise(mean = round(mean(sum), 0), sd = sd(sum)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Total&quot;)


b &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(81:83))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.81&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.82&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.83&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_1&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)


c &lt;- data_unique %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(&quot;Fraction.&quot;) | starts_with(&quot;name&quot;)) %&gt;% 
  select(matches(&quot;name&quot;) | contains(as.character(84:86))) %&gt;% 
  pivot_longer(!name, names_to = &quot;fraction&quot;, values_to = &quot;peptides&quot;) %&gt;% 
  mutate(peptides = case_when(is.na(peptides) ~ 0, TRUE ~ as.numeric(1))) %&gt;% 
  mutate(fraction = case_when(fraction == &quot;Fraction.84&quot; ~ &quot;1&quot;,
                              fraction == &quot;Fraction.85&quot; ~ &quot;2&quot;,
                              fraction == &quot;Fraction.86&quot; ~ &quot;3&quot;)) %&gt;%
  group_by(name) %&gt;% 
  mutate(unique = if_else(sum(peptides) == 1, T, F)) %&gt;% 
  mutate(unique_2 = if_else(peptides == 1 &amp; unique, fraction, &quot;no_unique&quot;)) %&gt;% 
  ungroup() %&gt;% 
  dplyr::count(unique_2, name = &quot;rep_2&quot;) %&gt;% 
  filter(unique_2 != &quot;no_unique&quot;)  
  
table &lt;- b %&gt;% 
  left_join(c) %&gt;%
  dplyr::rename(fraction = unique_2) %&gt;% 
  pivot_longer(!fraction, names_to = &quot;replicate&quot;, values_to = &quot;value&quot;) %&gt;% 
  group_by(fraction) %&gt;% 
  summarize(mean = round(mean(value), 0), sd = sd(value)) %&gt;% 
  ungroup() %&gt;% 
  mutate(Identity = &quot;Unique&quot;) %&gt;% 
  rbind(a, .)

table_perc &lt;- table %&gt;% 
  select(-sd) %&gt;% 
  pivot_wider(names_from = Identity, values_from = mean) %&gt;% 
  mutate(perc = paste(round(Unique * 100 / Total,0),&quot;%&quot;,sep=&quot;&quot;)) %&gt;% 
  pivot_longer(!contains(&quot;r&quot;), names_to = &quot;Identity&quot;, values_to = &quot;n_peptides&quot;) %&gt;% 
  mutate(perc = case_when(Identity == &quot;Total&quot; ~ as.character(n_peptides), TRUE ~ as.character(perc)))

p1 &lt;- ggplot(table, aes(x=fraction, y=mean, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge())+
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=.2, size=1, position=position_dodge(.9)) +
  geom_text(aes(label=mean), vjust=-1,position = position_dodge(0.9), size=13) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme_minimal()+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25))+
  theme_DEP1()

p2 &lt;- ggplot(table_perc, aes(x=fraction, y=n_peptides, fill=Identity)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge(0))+
  geom_text(aes(label=perc), vjust=-.4, size=12) +
  scale_y_continuous(n.breaks = 6, limits = c(0,3500))+  ## cambiare il limite in base al plot
  labs(x = &quot;Fractions&quot;, y = &quot;n° proteins&quot;)+
  theme(axis.text.x = element_text(size=35, angle=0),
        axis.text.y = element_text(size=30, angle=0),
        axis.title=element_text(size=35),
        legend.title = element_text(size = 30),
        legend.text = element_text(size = 25)) +
  theme_DEP1()

p1 / p2</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/n_proteins_SP3_23-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="fractionation-ist-comparisons" class="section level3 tabset tabset-fade">
<h3>Fractionation iST comparisons</h3>
<p>In this analysis I compare the different number of protein identify in the three fractionation methods in <strong>iST</strong>.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign_2.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(&quot;Fraz_8_iST_HpH&quot;, &quot;Fraz_6_iST_HpH&quot;, &quot;Fraz_3_iST_AddOn&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;Intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

Fractionation &lt;- make_se(data_unique, columns, expdesign)</code></pre>
<div id="protein-per-sapmle-3" class="section level4">
<h4>Protein per sapmle</h4>
<pre class="r"><code>plot_numbers_lables(Fractionation, sub_title = &quot;Fractionation iST&quot;, lab_size = 10)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_numb_4-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="distibution-of-each-sample-3" class="section level4">
<h4>Distibution of each sample</h4>
<pre class="r"><code>plot_normalization(Fractionation)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_norm_4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-fractionation-3-1" class="section level4">
<h4>Upset plot Fractionation 3</h4>
<pre class="r"><code>define_set &lt;- c(&quot;iST_50ug_HpH8_1&quot;, &quot;iST_50ug_HpH8_2&quot;, &quot;iST_50ug_HpH6_1&quot;, &quot;iST_50ug_HpH6_2&quot;, &quot;iST_50ug_AddOn_1&quot;, &quot;iST_50ug_AddOn_2&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;%
  upset(nsets = 6,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;turquoise3&quot;, &quot;forestgreen&quot;, &quot;coral2&quot;), each = 2),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/up_4-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-mean" class="section level4">
<h4>Upset plot mean</h4>
<pre class="r"><code>define_set &lt;- c(&quot;iST_50ug_HpH8&quot;, &quot;iST_50ug_HpH6&quot;, &quot;iST_50ug_AddOn&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;_.$&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  group_by(name, samples) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, 0, 1)) %&gt;% 
  ungroup() %&gt;% 
  select(-intensity) %&gt;% 
  group_by(name) %&gt;% 
  distinct() %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(id_cols = name, names_from = samples, values_from = unique) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = c(&quot;turquoise3&quot;, &quot;forestgreen&quot;, &quot;coral2&quot;),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/venn-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="venn-mean" class="section level4">
<h4>Venn mean</h4>
<pre class="r"><code>knitr::include_graphics(here::here(&quot;data/venn_iST.png&quot;))</code></pre>
<p><img src="C:/Users/ieo4973/Documents/Internal_test/data/venn_iST.png" width="640" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="fractionation-sp3-comparisons" class="section level3 tabset tabset-fade">
<h3>Fractionation SP3 comparisons</h3>
<p>In this analysis I compare the different number of protein identify in the three fractionation methods in <strong>SP3</strong>.</p>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign_2.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(&quot;Fraz_8_SP3_HpH&quot;, &quot;Fraz_6_SP3_HpH&quot;, &quot;Fraz_3_SP3_AddOn&quot;)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;Intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

Fractionation &lt;- make_se(data_unique, columns, expdesign)</code></pre>
<div id="protein-per-sapmle-4" class="section level4">
<h4>Protein per sapmle</h4>
<pre class="r"><code>plot_numbers_lables(Fractionation, sub_title = &quot;Fractionation SP3&quot;, lab_size = 10)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_numb_5-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="distibution-of-each-sample-4" class="section level4">
<h4>Distibution of each sample</h4>
<pre class="r"><code>plot_normalization(Fractionation)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/plot_norm_5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-fractionation-3-2" class="section level4">
<h4>Upset plot Fractionation 3</h4>
<pre class="r"><code>define_set &lt;- c(&quot;SP3_50ug_HpH8_1&quot;, &quot;SP3_50ug_HpH8_2&quot;, &quot;SP3_50ug_HpH6_1&quot;, &quot;SP3_50ug_HpH6_2&quot;, &quot;SP3_50ug_AddOn_1&quot;, &quot;SP3_50ug_AddOn_2&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 6,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;turquoise3&quot;, &quot;forestgreen&quot;, &quot;coral2&quot;), each = 2),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/up_5-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-mean-1" class="section level4">
<h4>Upset plot mean</h4>
<pre class="r"><code>define_set &lt;- c(&quot;SP3_50ug_HpH8&quot;, &quot;SP3_50ug_HpH6&quot;, &quot;SP3_50ug_AddOn&quot;)

data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;_.$&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  group_by(name, samples) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, 0, 1)) %&gt;% 
  ungroup() %&gt;% 
  select(-intensity) %&gt;% 
  group_by(name) %&gt;% 
  distinct() %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(id_cols = name, names_from = samples, values_from = unique) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        sets = define_set,
        #order.by = &quot;degree&quot;,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = c(&quot;turquoise3&quot;, &quot;forestgreen&quot;, &quot;coral2&quot;),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/venn2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="venn-mean-1" class="section level4">
<h4>Venn mean</h4>
<pre class="r"><code>knitr::include_graphics(here::here(&quot;data/venn_SP3.png&quot;))</code></pre>
<p><img src="C:/Users/ieo4973/Documents/Internal_test/data/venn_SP3.png" width="640" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="charges" class="section level3">
<h3>Charges</h3>
<pre class="r"><code>peptides_unique %&gt;% 
  as.data.frame() %&gt;%
  select(contains(&quot;charge&quot;)) %&gt;% 
  mutate(Charges = gsub(pattern = &quot;;.&quot;, replacement = &quot;&quot;, Charges)) %&gt;% 
  dplyr::count(Charges) %&gt;% 
  ggplot(mapping = aes(x = Charges, y = n, fill = Charges)) +
  geom_bar(stat=&quot;identity&quot;) +
  geom_text(aes(label=n), vjust=-.4, size=9) + 
  theme(axis.line = element_line(size = 0.3, linetype = &quot;solid&quot;), 
        axis.ticks = element_line(size = 1), 
        panel.grid.major = element_line(colour = &quot;gray95&quot;, size = 0.3), 
        panel.grid.minor = element_line(colour = &quot;gray95&quot;, size = 0.3), 
        axis.title = element_text(size = 18, face = &quot;bold&quot;, colour = &quot;black&quot;), 
        axis.text = element_text(size = 20, colour = &quot;gray15&quot;), 
        plot.title = element_text(size = 20, face = &quot;bold&quot;, hjust = 0.5), 
        legend.text = element_text(size = 14), 
        legend.title = element_text(size = 14, face = &quot;bold&quot;), 
        panel.background = element_rect(fill = &quot;white&quot;)) +
  labs(title = &quot;Charge distribution&quot;, y = &quot;Counts&quot;)</code></pre>
<p><img src="figure/comparison_iST_vs_SP3_fraz.Rmd/charge-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gene_name &lt;- data_unique %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(c(&quot;name&quot;, &quot;Intensity.&quot;))) %&gt;%
  ## define combination of interest in the original dataset
  select(starts_with(&quot;name&quot;) | contains(&quot;AddOn&quot;) | contains(&quot;HpH&quot;)) %&gt;% 
  pivot_longer(!name, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(intensity == 0, 0, 1)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;^.*\\.&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  mutate(samples = gsub(pattern = &quot;_.$&quot;, replacement = &quot;&quot;, samples)) %&gt;% 
  group_by(name, samples) %&gt;% 
  mutate(unique = if_else(sum(intensity) == 0, 0, 1)) %&gt;% 
  ungroup() %&gt;% 
  select(-intensity) %&gt;% 
  group_by(name) %&gt;% 
  distinct() %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(id_cols = name, names_from = samples, values_from = unique) %&gt;%
  mutate(gene_name = if_else(SP3_50ug_AddOn == 1, T, F)) %&gt;% 
  filter(gene_name) %&gt;% 
  pull(name)
#write.table(gene_name,file = &quot;gene_name_SP3_50ug_AddOn.txt&quot;, sep=&quot;\t&quot;, quote=FALSE, row.names=FALSE, col.names = F)</code></pre>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<hr />
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggrepel_0.9.1               UpSetR_1.4.0               
 [3] ggstatsplot_0.7.2           datapasta_3.1.0            
 [5] here_1.0.1                  patchwork_1.1.1            
 [7] stringr_1.4.0               DT_0.18                    
 [9] enrichR_3.0                 ggplot2_3.3.3              
[11] tibble_3.1.0                preprocessCore_1.52.1      
[13] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[15] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[17] IRanges_2.24.1              S4Vectors_0.28.1           
[19] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[21] matrixStats_0.58.0          DEP_1.12.0                 
[23] tidyr_1.1.3                 dplyr_1.0.5                
[25] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                shinydashboard_0.7.1     
  [3] gmm_1.6-6                 tidyselect_1.1.1         
  [5] RSQLite_2.2.5             htmlwidgets_1.5.3        
  [7] grid_4.0.3                BiocParallel_1.24.1      
  [9] gmp_0.6-2                 norm_1.0-9.5             
 [11] munsell_0.5.0             codetools_0.2-16         
 [13] effectsize_0.4.4-1        withr_2.4.2              
 [15] colorspace_2.0-0          highr_0.8                
 [17] knitr_1.31                ipmisc_6.0.2             
 [19] ggsignif_0.6.1            mzID_1.28.0              
 [21] labeling_0.4.2            git2r_0.28.0             
 [23] GenomeInfoDbData_1.2.4    farver_2.1.0             
 [25] bit64_4.0.5               rprojroot_2.0.2          
 [27] coda_0.19-4               vctrs_0.3.7              
 [29] generics_0.1.0            xfun_0.23                
 [31] BiocFileCache_1.14.0      BWStest_0.2.2            
 [33] R6_2.5.0                  doParallel_1.0.16        
 [35] clue_0.3-58               BayesFactor_0.9.12-4.2   
 [37] bitops_1.0-6              cachem_1.0.4             
 [39] reshape_0.8.8             DelayedArray_0.16.2      
 [41] assertthat_0.2.1          promises_1.2.0.1         
 [43] scales_1.1.1              gtable_0.3.0             
 [45] Cairo_1.5-12.2            affy_1.68.0              
 [47] multcompView_0.1-8        sandwich_3.0-0           
 [49] rlang_0.4.10              MatrixModels_0.5-0       
 [51] zeallot_0.1.0             mzR_2.24.1               
 [53] GlobalOptions_0.1.2       PMCMRplus_1.9.0          
 [55] impute_1.64.0             BiocManager_1.30.10      
 [57] yaml_2.2.1                httpuv_1.5.5             
 [59] tools_4.0.3               affyio_1.60.0            
 [61] ellipsis_0.3.1            jquerylib_0.1.3          
 [63] RColorBrewer_1.1-2        WRS2_1.1-1               
 [65] MSnbase_2.15.7            Rcpp_1.0.6               
 [67] plyr_1.8.6                zlibbioc_1.36.0          
 [69] purrr_0.3.4               RCurl_1.98-1.2           
 [71] pbapply_1.4-3             GetoptLong_1.0.5         
 [73] correlation_0.6.1         zoo_1.8-9                
 [75] cluster_2.1.0             fs_1.5.0                 
 [77] magrittr_2.0.1            circlize_0.4.12          
 [79] pcaMethods_1.82.0         mvtnorm_1.1-1            
 [81] whisker_0.4               ProtGenerics_1.22.0      
 [83] hms_1.0.0                 mime_0.10                
 [85] evaluate_0.14             xtable_1.8-4             
 [87] XML_3.99-0.6              pairwiseComparisons_3.1.5
 [89] gridExtra_2.3             shape_1.4.5              
 [91] compiler_4.0.3            ncdf4_1.17               
 [93] crayon_1.4.1              htmltools_0.5.1.1        
 [95] mc2d_0.1-19               later_1.1.0.1            
 [97] DBI_1.1.1                 SuppDists_1.1-9.5        
 [99] kSamples_1.2-9            dbplyr_2.1.1             
[101] ComplexHeatmap_2.6.2      MASS_7.3-53              
[103] tmvtnorm_1.4-10           rappdirs_0.3.3           
[105] Matrix_1.2-18             readr_1.4.0              
[107] vsn_3.58.0                imputeLCMD_2.0           
[109] insight_0.14.0            pkgconfig_2.0.3          
[111] statsExpressions_1.0.1    MALDIquant_1.19.3        
[113] paletteer_1.3.0           foreach_1.5.1            
[115] bslib_0.2.4               ggcorrplot_0.1.3         
[117] XVector_0.30.0            digest_0.6.27            
[119] parameters_0.13.0         rmarkdown_2.7            
[121] curl_4.3                  shiny_1.6.0              
[123] gtools_3.8.2              rjson_0.2.20             
[125] lifecycle_1.0.0           jsonlite_1.7.2           
[127] limma_3.46.0              fansi_0.4.2              
[129] pillar_1.6.0              lattice_0.20-41          
[131] fastmap_1.1.0             httr_1.4.2               
[133] glue_1.4.2                bayestestR_0.9.0         
[135] png_0.1-7                 iterators_1.0.13         
[137] bit_4.0.4                 stringi_1.5.3            
[139] sass_0.3.1                performance_0.7.1        
[141] rematch2_2.1.2            blob_1.2.1               
[143] memoise_2.0.0             Rmpfr_0.8-4              </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
